java/Makefile.docker

```diff
@@ -2,8 +2,8 @@
 #
 
 # Docker tests variables
-DOCKER_CONTAINER_BASE = systemsmanagement/uyuni/master/docker/containers/uyuni-master
-DOCKER_REGISTRY       = registry.opensuse.org
+DOCKER_CONTAINER_BASE = devel/galaxy/manager/4.3/docker/containers/suma-4.3
+DOCKER_REGISTRY       = registry.suse.de
```

Comment:
This is not for uyuni, right?

Reply:
oh right... I forgot to drop it in cherry-picking from 4.3

Reply:
fixed